### DISCUSSION TITLE
Extract entry module with rollup manual chunks

### DISCUSSION THREAD

**Discussion by oleghalin** [nuxt/nuxt#16624] [Questions]
Use case:
I want to extract @sentry module from my entry chunk

I tried rollupOptions.manualChunks but bundler says next: 
`Entry module cannot be external`

**Comment by TheAlexLichter** [comment]
Could you provide a StackBlitz/CodeSandbox of what you tried so far?

**Reply by oleghalin** [reply]
Sure
https://stackblitz.com/edit/github-9ukebr-q4gqxy?file=nuxt.config.ts
<img width="439" alt="image" src="https://user-images.githubusercontent.com/9086983/195111078-6132a7e4-d6ac-4f2f-944f-87bbb6620620.png">

**Reply by oleghalin** [reply]
Resoled it with dynamic import in plugin, some sort of duck coding, thanks :D 

Maybe it could be useful for someone

```
const Sentry = await import("@sentry/vue");
const { BrowserTracing } = await import("@sentry/tracing");
```

**Reply by richardkenneth10** [reply]
How can I set the rollupOptions.manualChunks option for production in Nuxt3 because I learnt that vite is used for development

**Reply by praveshishere** [reply]
chunks are usually meant for production only, and roll-up is required only for the build. So you don't need to do anything else, it will just work.
